docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
 
 .. todo::
 
-   Format as table with measure, measure definition, data sources and
-   their uses.
+   Update mortality-related data sources within existing format.
+
+Once a cause model structure is specified, data is needed to inform its states
+and transitions. For our purposes, cause models generally have the following
+data needs:
+
+#. The probability that a simulant will start the simulation in a given state
+   within the cause model.
+#. The probability that a simulant will transition to a new state within the
+   cause model in a given time-step.
+#. The disability weight for each state in the cause model
+#. The probability that a simulant in a given cause model state will die in a
+   given time-step.
+
+There are several common data sources that can be used for these needs, which
+are outlined in the table below and discussed in more detail afterward.
+`Non-standard data sources`_ are discussed later on this page.
```

Comment:
I don't think it's necessarily worthwhile to split standard and non-standard data sources into two sections.